assets/javascripts/feature.js

```diff
@@ -1,79 +1,130 @@
-// jshint esversion: 6
+var checkCheckbox = false;
+
+function postFeature(latestFeature) {
+  $.ajax({
+    url: '/api/v1/feature',
+    method: 'POST',
+    data: {version: checkCheckbox ? 0 : latestFeature}
+  });
+}
+
+function switchCheckbox(checkboxElement) {
+  if (checkboxElement.checked) {
+    checkCheckbox = true;
+  } else {
+    checkCheckbox = false;
+  }
```

Comment:
```suggestion
  checkCheckbox = checkboxElement.checked;
```

Although I would just inline it completely, see the other comment.